plugin/noop.go

```diff
@@ -0,0 +1,166 @@
+package plugin
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"net/url"
+	"os"
+	"strings"
+	"sync"
+
+	"github.com/hashicorp/go-hclog"
+
+	"github.com/connylabs/ingest"
+	"github.com/connylabs/ingest/storage"
+)
+
+var (
+	// DefaultCodec can be used for testing against this noop plugin.
+	DefaultCodec  = ingest.SimpleCodec{XID: "id", XName: "name"}
+	DefaultLogger = hclog.New(&hclog.LoggerOptions{
+		Level:      hclog.Trace,
+		Output:     os.Stderr,
+		JSONFormat: true,
+	})
+)
+
+const (
+	DefaultObjContent = "content of the default object"
```

Comment:
These should probably all be unexported, right?
```suggestion
	defaultObjContent = "content of the default object"
```